src/java.base/share/classes/java/lang/Object.java

```diff
@@ -214,13 +226,15 @@ public boolean equals(Object obj) {
     protected native Object clone() throws CloneNotSupportedException;
 
     /**
-     * Returns a string representation of the object. In general, the
+     * Returns a string representation of the object.
+     * @apiNote
+     * In general, the
      * {@code toString} method returns a string that
      * "textually represents" this object. The result should
      * be a concise but informative representation that is easy for a
      * person to read.
      * It is recommended that all subclasses override this method.
```

Comment:
Do we want to have a general note here or somewhere that the exact format of the result of `toString()` is generally not stable, and that it is subject to change without notice?

Reply:
Added "The string output is not necessary stable over time." The particulars of the toString contract of a given class will depend on the class of course.

Reply:
Perhaps "necessary" -> "necessarily"

Reply:
Good catch Roger; will change to
"The string output is not necessarily stable over time or across JVM invocations."
The latter clause will more explicitly handle cases like "myEnumConstant.toString() gave different answers when I ran the program multiple times," a bug that has been submitted as closed as not-a-bug repeated.